### PR TITLE
Update Utah ICU Beds Number

### PR DESCRIPTION
**Pull request by BrettBoval**
According to the Utah Department of Health, they only have possible staff for 85% of the ICU bed listed.

Note about ICU Utilization: The UDOH receives reports about the total number of available ICU beds in the state. However, this reporting does not reflect the number of staffed beds able to care for patients. At about 69% overall ICU utilization, ICUs in Utah’s major hospitals with the ability to provide best care for COVID-19 patients begin to reach staffing capacity. Seventy-two percent use creates major strains on the healthcare system. When 85% capacity is reached, Utah will be functionally out of staffed ICU beds, indicating an overwhelmed hospital system. Please see the Unified Response Scoreboard to see these measures with thresholds over the past calendar week.

round(664*0.85) = 564

https://trello.com/c/u8surBE7/533-small-icu-headroom-update-update-utah-decompensation-factor-to-align-with-local-sources

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -4,7 +4,8 @@ set -o errexit
 
 python scripts/update.py
 python scripts/update_covid_tracking_data.py
-python scripts/update_covid_care_map.py
+# TODO(brett): Change CCM to static file from 2018 Hospital Survey
+# python scripts/update_covid_care_map.py
```
Comment (ghop02) [inline update.sh:8]: hmmm I would rather do this in the updater itself. Even if we're not updating the file, I think it's dangerous that running the updater will erase that change.  Also makes it easier to add in the comment you had in the PR to the code explaining why that number is different

Comment (BrettBoval) [reply update.sh:8]: In the medium term I'd like to switch this CCM to a static source for the 2018 pre-covid hospitalization document plus a diff log of changes we've done to update it. They've started making their latest endpoint a mix of pre and mid-covid changes that we don't want. So it'll be part of the bigger change, that depends on the future of the ICU metric.

Comment (TomGoBravo) [reply update.sh:8]: I'm looking to change how the covid-data-model repo loads CCM data. Are you saying we give up on getting future updates from CCM and restrict updating this data to people who can modify a CSV in github?

**Comment by BrettBoval** [comment]
Correct. We no longer want to incorporate changes made upstream by CCM.
They have changed the mandate of their dataset into a hybrid of historical
and current data in a way that is not useful for us. If we maintain the ICU
utilization going forward we will prioritize the HHS hospital data, and may
draw static capacity values for counties.

**Pull request merged**